README.md

```diff
@@ -244,6 +245,28 @@ authClient.verifyRecoveryToken({
 });
 ```
 
+## [webfinger(options)]
+
+Calls the Webfinger API and gets a response.
+
+- `resource` - URI that identifies the entity whose information is sought, currently only acct scheme is supported (e.g acct:dade.murphy@example.com)
+- `rel` - Optional parameter to request only a subset of the information that would otherwise be returned without the "rel" parameter
+- `requestContext` - Optional parameter that provides Webfinger the context of that which the user is trying to access, such as the path of an app
+
+```javascript
+authClient.webfinger({
+  resource: 'acct:john.joe@example.com',
```

Comment:
do we want to allow a custom issuer or is webfinger only available on the root Okta url?